hack/boilerplate/boilerplate.py

```diff
@@ -133,6 +133,9 @@ def file_extension(filename):
 skipped_dirs = ['Godeps', 'third_party', '_gopath', '_output', '.git', 'cluster/env.sh',
                 "vendor", "test/e2e/generated/bindata.go", "hack/boilerplate/test"]
 
+# Skip event-adapter, which is not authored by Google Inc.
```

Comment:
It's based on this framework: https://github.com/kubernetes-incubator/custom-metrics-apiserver (precisely on some old version of it, btw. I think it should be merged at some point). The framework is authored by "The Kubernetes Authors", and the boilerplate check requires it to say it's "Google Inc"